src/dotty/tools/dotc/core/Types.scala

```diff
@@ -2082,6 +2082,13 @@ object Types {
       if (parent.member(refinedName).exists) derivedRefinedType(parent, refinedName, refinedInfo)
       else parent
 
+    // MemberInfo methods
+    def exists(implicit ctx: Context) = true
```

Comment:
Does it mean that a Refinement to ErrorType\NoType `exists`? Shouldn't there be some sanity check here?